modules/imaging_browser/README.md

```diff
@@ -78,7 +78,10 @@ mantis_url - This setting defines a URL for LORIS to include a link to for bug r
   `$LORIS/tools/CouchDB_Import_MRI.php` alongside all other CouchDB
   import scripts, but should logically be considered part of this module.)
 - The imaging browser module includes links to BrainBrowser to visualize data.
+- The control panel on the viewsession page includes links to instruments
+  named "mri_parameter_form" and "radiologyreview" if they exist for the
+  currently viewed session.
 - The control panel on the viewsession page includes links to the instruments
   as configured by the study.
-- The control panel on the viewsession page includes links to the DICOM Archive
-  for any DICOM tars associated with the given session.
+- The control panel on the viewsession page includes links to the DICOM Archive 
```

Comment:
Same here, "View Session" instead of viewsession

Reply:
done